### PR TITLE
Table with Fig TFE Links

### PR DESCRIPTION
**Pull request by chantelleleveille**
# Goal:
Add markdown table containing links for each figure and a direct link to explore the result in TFE. 

Point to it from the main readme

Preview:
Screen Shot 2024-11-15 at 10 48 11 AM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,3 +70,7 @@ python nuc_morph_analysis/lib/preprocessing/generate_perturbation_manifest.py
 ```bash
 python nuc_morph_analysis/lib/visualization/write_data_for_colorizer.py
 ```
+
+## Explore results in timelapse feature explorer
+
+To interactively explore the results and features generated in this repository and shown in each figure panel of the paper follow the links provided in [this table](/docs/TFE_links.md).
```
Comment (jcass11) [inline README.md]: each figure panel might be a slight over-promise relative to what I included. 😜

Comment (jcass11) [inline README.md]:
```suggestion
To interactively explore the results and features generated in this repository and shown in the paper, follow the links provided in [this table](/docs/TFE_links.md). 
```

File: docs/TFE_links.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Timelapse feature explorer links for data shown in figures
+
+[Timelapse Feature Explorer](https://timelapse.allencell.org/) is a web-based tool that allows users to view timelapse segmentations with interactive and customizable colorization mapped according to relative intensities of quantitative features. In this viewer, nuclear segmentations from each individual colony timelapse are colorized by a chosen feature, with each nucleus in each frame being colored by its feature value in that frame. The resulting timelapses demonstrate how growth and shape features vary for all nuclei across the colony and over time, allowing for a qualitative assessment of spatiotemoral variation of these features. The tool also allows for selection of single nuclear trajectories, to visualize how individuals behave over time and compare to the population.
```
Comment (jcass11) [inline docs/TFE_links.md]: Nit: What do you think of "relative magnitudes of quantitative features"? Not sure intensities is quiiiiiiiite right.

File: docs/TFE_links.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Timelapse feature explorer links for data shown in figures
+
+[Timelapse Feature Explorer](https://timelapse.allencell.org/) is a web-based tool that allows users to view timelapse segmentations with interactive and customizable colorization mapped according to relative intensities of quantitative features. In this viewer, nuclear segmentations from each individual colony timelapse are colorized by a chosen feature, with each nucleus in each frame being colored by its feature value in that frame. The resulting timelapses demonstrate how growth and shape features vary for all nuclei across the colony and over time, allowing for a qualitative assessment of spatiotemoral variation of these features. The tool also allows for selection of single nuclear trajectories, to visualize how individuals behave over time and compare to the population. 
+
+To explore the results and features generated by repository and shown in each figure panel of the paper Dixon et al. 2024 [(bioRxiv)](https://www.biorxiv.org/content/10.1101/2024.06.28.601071v1) follow the links in the table below.
```
Comment (jcass11) [inline docs/TFE_links.md]:
```suggestion
To explore the results and features generated by repository and shown in the paper Dixon et al. 2024 [(bioRxiv)](https://www.biorxiv.org/content/10.1101/2024.06.28.601071v1) follow the links in the table below. 
```

**Comment by jcass11** [comment]
Yay looks amazing! Thanks for taking the action to transfer the spreadsheet into a beautiful markdown table with documentation. I'm so excited about it!

**Review by jcass11 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chantelleleveille) [reply README.md]: You got it!!

File: docs/TFE_links.md (hunk 2 above)
Comment (chantelleleveille) [reply docs/TFE_links.md]: Sure :)

**Comment by jcass11** [comment]
@cfrick13 on the google sheet version I made (its a tab on the nuc morph to-do spreadheer) I included placeholders for including the example trajectories from your S10 dip figure. Once those features are added to public TFE we can move those into this markdown table too!

**Review by pgarrison — APPROVED** [review]
Super! I tested all the links in the table and they all load things in TFE successfully. Also the link from the README.md to the TFE_links.md works

**Pull request merged**